### PR TITLE
First draft of available sensors' documentation 

### PR DESCRIPTION
**Pull request by mfussi66**
This PR begins addressing https://github.com/icub-tech-iit/documentation/issues/271, by taking inspiration from the OP.
The idea is to have a dedicated page where each type of sensor (controlboard included) has their own server and client briefly described, with links to the YARP docs. Then, for each, robot, a list of the mounted sensors and basic info on how to access their data.

### CODE REVIEW

File: docs/icub_sensors/icub27_sensors.md
[the remark this answers is not in the document]
Comment (pattacini) [reply docs/icub_sensors/icub27_sensors.md:1]: Or even `icub2.7_`

[the remark this answers is not in the document]
Comment (mfussi66) [reply docs/icub_sensors/icub27_sensors.md:1]: I guess `icub2.7_` could be good! 2x_ might be too general.

File: docs/icub_sensors/yarp_nws_convention.md [hunk 1]
```diff
@@ -108,4 +112,11 @@ The `RgbdSensor_nws_yarp` device will open the following YARP ports:
 - `/<robotPortPrefix>/<cameraName>/depth:o` : Publishes the camera's depth image as a list of floating point values
 - `/<robotPortPrefix>/<cameraName>/rpc` : Exposes several information related to the part via a YARP RPC port
 
-**Note: At the moment no client device is available for depth cameras, so the server needs to be accessed directly from the above ports.**
\ No newline at end of file
+#### RGBD Camera client device
+
+**Note: At the moment no client device is available for depth cameras, so the server needs to be accessed directly from the above ports.**
+
```
Comment (pattacini) [inline docs/icub_sensors/yarp_nws_convention.md:118]: Use [mkdocs admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) instead

**Review by pattacini — CHANGES_REQUESTED** [review]
Better off using mkdocs admonitions.
See inline comments.

File: docs/icub_sensors/yarp_nws_convention.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+The `RgbdSensor_nws_yarp` device will open the following YARP ports:
+
+- `/<robotPortPrefix>/<cameraName>/image:o`: Publishes the camera's RGB image
+- `/<robotPortPrefix>/<cameraName>/depth:o` : Publishes the camera's depth image as a list of floating point values
+- `/<robotPortPrefix>/<cameraName>/rpc` : Exposes several information related to the part via a YARP RPC port
+
+#### RGBD Camera client device
+
+**Note: At the moment no client device is available for depth cameras, so the server needs to be accessed directly from the above ports.**
```
Comment (pattacini) [inline docs/icub_sensors/yarp_nws_convention.md:117]: Use [mkdocs admonitions](https://squidfunk.github.io/mkdocs-material/reference/admonitions/#usage) instead

File: mkdocs.yml [hunk 3]
```diff
@@ -271,6 +271,7 @@ nav:
         - Configure IP on a setup for ETH boards: configure_static_ip/configure-static-ip.md
       - Using iCub with ROS: icub_ros/index.md
       - iCub Force Control: icub_force_control/icub-force-control.md
+      - Sensor interfaces: icub_sensors/index.md
```
Comment (pattacini) [inline mkdocs.yml:274]: Nitpicking, but we could replicate here the hierarchy with sublevels, as done with the other resources.

**Review by pattacini — CHANGES_REQUESTED** [review]
One last comment to address from my side before merging this iteration.

**Review by pattacini — CHANGES_REQUESTED** [review]
Since we deal with a variety of robots here, why don't we change references to `icub_sensors` into `robot_sensors`? This is just to remain a little bit more generic.

File: mkdocs.yml (hunk 3 above)
Comment (mfussi66) [reply mkdocs.yml]: Done!

**Review by pattacini — APPROVED** [review]

**Pull request merged**